<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainSkel</class>
 <widget class="QMainWindow" name="mainSkel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HotSpotter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="root_hlayout">
      <item>
       <widget class="QTabWidget" name="tablesTabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="image_view">
         <property name="minimumSize">
          <size>
           <width>445</width>
           <height>0</height>
          </size>
         </property>
         <attribute name="title">
          <string>Image Table</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTableWidget" name="image_TBL">
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="chip_view">
         <attribute name="title">
          <string>Chip Table</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTableWidget" name="chip_TBL">
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="result_view">
         <attribute name="title">
          <string>Results Table</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTableWidget" name="res_TBL">
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="status_HLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Database"/>
    <addaction name="actionOpen_Database"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Database"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Img_file"/>
    <addaction name="actionImport_Img_dir"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionLayout_Figures"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionView_Docs"/>
    <addaction name="separator"/>
    <addaction name="actionView_DBDir"/>
    <addaction name="actionView_Computed_Dir"/>
    <addaction name="actionView_Global_Dir"/>
    <addaction name="separator"/>
    <addaction name="actionWriteLogs"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_computed_directory"/>
    <addaction name="actionDelete_global_preferences"/>
    <addaction name="separator"/>
    <addaction name="actionDev_Mode_IPython"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionAdd_Chip"/>
    <addaction name="actionNew_Chip_Property"/>
    <addaction name="separator"/>
    <addaction name="actionQuery"/>
    <addaction name="separator"/>
    <addaction name="actionReselect_ROI"/>
    <addaction name="actionReselect_Ori"/>
    <addaction name="separator"/>
    <addaction name="actionNext"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_Chip"/>
   </widget>
   <widget class="QMenu" name="menuBatch">
    <property name="title">
     <string>Batch</string>
    </property>
    <addaction name="actionPrecomputeChipsFeatures"/>
    <addaction name="actionPrecompute_Queries"/>
    <addaction name="separator"/>
    <addaction name="actionScale_all_ROIS"/>
    <addaction name="separator"/>
    <addaction name="actionConvert_all_images_into_chips"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menuBatch"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_Database">
   <property name="text">
    <string>Open Database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSave_Database">
   <property name="text">
    <string>Save Database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionImport_Img_file">
   <property name="text">
    <string>Import Images (select file)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionOpen_Data_Directory">
   <property name="text">
    <string>View Data Directory</string>
   </property>
  </action>
  <action name="actionOpen_Source_Directory">
   <property name="text">
    <string>View Source Directory</string>
   </property>
  </action>
  <action name="actionTogEll">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle Ellipses</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionUndockDisplay">
   <property name="text">
    <string>Undock Display</string>
   </property>
  </action>
  <action name="actionTogPlt">
   <property name="text">
    <string>Toggle PlotWidget</string>
   </property>
  </action>
  <action name="actionHelpCMD">
   <property name="text">
    <string>Command Line Help</string>
   </property>
  </action>
  <action name="actionHelpGUI">
   <property name="text">
    <string>GUI Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionHelpWorkflow">
   <property name="text">
    <string>Workflow Help</string>
   </property>
  </action>
  <action name="actionTogPts">
   <property name="text">
    <string>Toggle Points</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionAdd_Chip">
   <property name="text">
    <string>Add Chip</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionReselect_ROI">
   <property name="text">
    <string>Reselect ROI</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Select Next</string>
   </property>
   <property name="toolTip">
    <string>Selects the next unidentified CID or Untagged GID</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionDelete_Chip">
   <property name="text">
    <string>Delete Chip</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionQuery">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Query</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionPrev">
   <property name="text">
    <string>Prev</string>
   </property>
  </action>
  <action name="actionWriteLogs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Write Logs</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionOpen_Internal_Directory">
   <property name="text">
    <string>View Internal Directory</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Edit Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionConvertImage2Chip">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Convert All Images to Chips</string>
   </property>
  </action>
  <action name="actionBatch_Change_Name">
   <property name="text">
    <string>Batch Change Name</string>
   </property>
  </action>
  <action name="actionReselect_Ori">
   <property name="text">
    <string>Reselect Orientation</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionAdd_Metadata_Property">
   <property name="text">
    <string>Add Chip Property</string>
   </property>
  </action>
  <action name="actionAutoassign">
   <property name="text">
    <string>Assign Matches Above Threshold</string>
   </property>
  </action>
  <action name="actionRankErrorExpt">
   <property name="text">
    <string>Rank Error Experiment</string>
   </property>
  </action>
  <action name="actionName_Consistency_Experiment">
   <property name="text">
    <string>Name Consistency</string>
   </property>
  </action>
  <action name="actionIncrease_ROI_Size">
   <property name="text">
    <string>Increase all ROI Sizes</string>
   </property>
  </action>
  <action name="actionView_Docs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View Documentation</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionAutoassign_2">
   <property name="text">
    <string>Auto-Assign Matches</string>
   </property>
  </action>
  <action name="actionNew_Database">
   <property name="text">
    <string>New Database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionImport_Img_dir">
   <property name="text">
    <string>Import Images (select directory)</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionPrecompute_Queries">
   <property name="text">
    <string>Precompute Queries</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionPrecomputeChipsFeatures">
   <property name="text">
    <string>Precompute Chips/Features</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionView_DBDir">
   <property name="text">
    <string>View Database Dir</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionScale_all_ROIS">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Scale all ROIs</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionConvert_all_images_into_chips">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Convert all images into chips</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionNew_Chip_Property">
   <property name="text">
    <string>New Chip Property</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionDelete_computed_directory">
   <property name="text">
    <string>Delete Computed Dir</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionDelete_global_preferences">
   <property name="text">
    <string>Delete Global Prefs</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionView_Computed_Dir">
   <property name="text">
    <string>View Computed Dir</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionView_Global_Dir">
   <property name="text">
    <string>View Global Dir</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionLayout_Figures">
   <property name="text">
    <string>Layout Figures</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionDev_Mode_IPython">
   <property name="text">
    <string>Developer Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Shift+D</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
